### ISSUE TITLE
 Clean project with cassandra-quarkus-client gets stuck in tests

### ISSUE THREAD

**Issue by rsvoboda** [datastax/cassandra-quarkus#132]
Clean project with cassandra-quarkus-client gets stuck in tests
There is just `[INFO] Running org.acme.getting.started.GreetingResourceTest` and nothing else.
I suspect it tries to connect to cassandra but there is no connection timeout.

After I hit Ctrl+C following message is printed
```
2020-09-02 15:43:49,246 INFO  [com.dat.oss.dri.int.cor.DefaultMavenCoordinates] (main) DataStax Java driver for Apache Cassandra(R) (com.datastax.oss:java-driver-core) version 4.7.2
2020-09-02 15:43:50,665 INFO  [com.dat.oss.dri.int.cor.tim.Clock] (vert.x-eventloop-thread-0) Using native clock for microsecond precision
```

Reproducer:
```
mvn io.quarkus:quarkus-maven-plugin:1.7.1.Final:create \
    -DprojectGroupId=org.acme \
    -DprojectArtifactId=getting-started \
    -DclassName="org.acme.getting.started.GreetingResource" \
    -Dpath="/hello" \
    -Dextensions="cassandra-quarkus-client"
cd getting-started
mvn package
```

**Comment by tomekl007** [comment]
Hello, the last `cassandra-quarkus-client` version : `1.0.0-alpha4-SNAPSHOT` supports `quarkus-platform` version `1.5.0.Final`. 
We are currently working on the next release, in which there will be support for `1.7.1.Final` (for more details see: https://github.com/datastax/cassandra-quarkus/issues/128)

**Comment by adutra** [comment]
Problem is likely solved by #148, at least I don't get any errors when executing the reproducer steps above.

**Comment by adutra** [comment]
Closing as this is likely solved by #148. Feel free to re-open if you think otherwise. Thanks!
